sql/planbuilder/scalar.go

```diff
@@ -347,11 +347,18 @@ func (b *PlanBuilder) buildBinaryScalar(inScope *scope, be *ast.BinaryExpr) sql.
 		default:
 			return expression.NewArithmetic(l, r, be.Operator)
 		}
-	case
-		ast.JSONExtractOp,
-		ast.JSONUnquoteExtractOp:
-		err := sql.ErrUnsupportedFeature.New(fmt.Sprintf("(%s) JSON operators not supported", be.Operator))
-		b.handleErr(err)
+
+	case ast.JSONExtractOp, ast.JSONUnquoteExtractOp:
```

Comment:
A plan/explain test to see that the operators are being converted properly could be nice.